lib/features/push_notification/presentation/listener/email_change_listener.dart

```diff
@@ -107,6 +107,10 @@ class EmailChangeListener extends ChangeListener {
         }
       } else if (action is PushNotificationAction) {
         _pushNotificationAction(action.newState, action.accountId, action.userName, action.session);
+
+        if (FcmUtils.instance.isMobileAndroid) {
+          _getNewReceiveEmailFromNotificationAction(action.session, action.accountId, action.newState);
+        }
```

Comment:
Can we do these actions in parallel?

Reply:
Processing inside the two functions is done, it doesn't have to wait for each other. It's just that it's being called in a synchronous function.